java/app-encryption/src/main/java/com/godaddy/asherah/appencryption/persistence/Metastore.java

```diff
@@ -30,4 +30,11 @@
    */
   boolean store(String keyId, Instant created, V value);
 
+  /**
+   * Returns the region suffix or "" if region suffix option is disabled
+   * @return The region suffix
+   */
+  default String getRegionSuffix() {
+    return "";
+  }
```

Comment:
Hmm, modifying the metastore interface was something I intentionally avoided with the Go implementation, though I suppose that's not as easy to avoid here. Shouldn't be an issue as the default method ensures backward compatibility... just thinking out loud :)